composer.json

```diff
@@ -27,16 +27,20 @@
         "classmap": ["src/Omnipay/Omnipay.php"]
     },
     "require": {
-        "php": ">=5.3.2",
-        "guzzle/guzzle": "~3.9",
-        "symfony/http-foundation": "~2.1"
+        "php": ">=5.5.9",
+        "symfony/http-foundation": "^2.1|^3.0",
```

Comment:
As you are using psr/http-message, wondering what symfony/http-foundation does. I am seeing it is also removed from code. May be you missed to remove it ?

Thank you.